src/core/Slider/component.tsx

```diff
@@ -1,110 +1,109 @@
-import React, { CSSProperties, ReactNode, useEffect, useRef } from "react";
-
+import React, { useState, useEffect, useRef, ReactNode } from "react";
 import Icon from "../Icon/component.tsx";
-import SliderScripts from "./component.js";
 import "./component.css";
 
-type SliderProps = {
-  slides?: ReactNode[];
-  classes?: string;
-  slideClasses?: string;
-  slideMinWidth?: string;
-  slideMaxWidth?: string;
-  mqEnableThreshold?: () => boolean;
+interface SliderProps {
+  children: ReactNode[];
+  interval?: number;
+}
 
-  container?: HTMLDivElement | null;
-};
+function SlideIndicator({
+  numSlides,
+  activeIndex,
+  interval,
+}: {
+  numSlides: number;
+  activeIndex: number;
+  interval: number;
+}) {
+  return (
+    <div className="flex gap-4 absolute -bottom-40 left-1/2 transform -translate-x-1/2">
+      {Array.from({ length: numSlides }, (_, i) => (
+        <div
+          key={i}
+          className={"relative w-40 h-4 mx-1 rounded-full bg-neutral-500"}
+        >
+          {i === activeIndex && (
+            <div
+              className="absolute inset-0 rounded-full bg-active-orange animate-[fillAnimation]"
+              style={{
+                animationDuration: `${interval}ms`,
```

Comment:
These inline properties might not be needed - it seems we can add to the `animate` Tailwind class above with further animation details: https://github.com/tailwindlabs/tailwindcss/discussions/9096#discussioncomment-3392101

Reply:
Tried it, did not really went well, but yeah, I can give it one more try